client/src/app/metadata-schema-form/custom/multiple-select/multiple-select.component.ts

```diff
@@ -82,6 +87,14 @@ export class MultipleSelectComponent implements OnInit {
     this.searchControl.valueChanges.subscribe(val => {
       this.onSearchValueChanged(val ? val : '');
     });
+
+    //TODO form data
```

Comment:
Looks like comment is no longer needed. Do you need to cleanup the todo comment ? or does it mean something else?

Reply:
There's quite a bit of work to resolve this involving a restructure of the base affected components. I'd like to merge the simpler changes first before working on this.